osidb/models.py

```diff
@@ -1138,6 +1157,33 @@ def bzsync(self, *args, bz_api_key, **kwargs):
         fc = FlawCollector()
         fc.sync_flaw(self.bz_id)
 
+    def tasksync(self, *args, jira_token, **kwargs):
```

Comment:
This method signature is weird, why is `*args` before `jira_token`? There's also no unit test for this method so I don't know what a call to it looks like or should look like.

Reply:
I have opted to follow `bzsync` signature, but I agree, I will change `tasksync`.